cmd/wallet-sdk-gomobile/wrapper/credentialreader.go

```diff
@@ -26,45 +23,25 @@ type CredentialReaderWrapper struct {
 
 // Get wraps Get of api.CredentialReader.
 func (r *CredentialReaderWrapper) Get(id string) (*verifiable.Credential, error) {
-	content, err := r.CredentialReader.Get(id)
+	vc, err := r.CredentialReader.Get(id)
 	if err != nil {
 		return nil, err
 	}
 
-	cred, err := verifiable.ParseCredential(content.Data, verifiable.WithDisabledProofCheck(),
-		verifiable.WithJSONLDDocumentLoader(r.DocumentLoader))
```

Comment:
if DocumentLoader no used any where else in CredentialReaderWrapper we can delete it

Reply:
Good catch! I'll make a followup PR shortly.